docker/bin/zookeeperStart.sh

```diff
@@ -51,6 +51,22 @@ if [ -f $DYNCONFIG ]; then
   ONDISK_DYN_CONFIG=true
 fi
 
+# Check if envoy is up and running
+if [[ -n "$ENVOY_SIDECAR_STATUS" ]]; then
```

Comment:
The code inside this `If block` shoudl get skipped  if environment variable "ENVOY_SIDECAR_STATUS" is **not** set. 
However, in generators.go we always  set this environment variable, that would cause this block to get executed on non-Istio environments as well.... 
Instead it may be good to inject any required environment variables using a config map, the name of which can be provided in the Zookeeper Spec. 
Similar to : https://github.com/pravega/bookkeeper-operator/blob/master/pkg/apis/bookkeeper/v1alpha1/bookkeepercluster_types.go#L142